specification/schemaregistry/SchemaRegistry/main.tsp

```diff
@@ -248,7 +248,7 @@ model SchemaGroups {
   /** The link to the next page of items */
   #suppress "@azure-tools/typespec-azure-core/casing-style" "Already public casing from service is not camelCase."
   @nextLink
-  NextLink?: string;
+  NextLink?: url;
```

Comment:
@hmlam @nickghardwick can you confirm that the next link returned from the service is an absolute URL?